Packs/CVE_2021_44228/Playbooks/playbook-CVE-2021-44228_-_Log4j_RCE.yml

```diff
@@ -2192,10 +2216,10 @@ tasks:
     isautoswitchedtoquietmode: false
   "60":
     id: "60"
-    taskid: 63813eaf-9b18-4965-8da5-0f928389863f
+    taskid: 8ea51d16-1d64-40e6-8d3a-d5a459ffaf70
     type: playbook
     task:
-      id: 63813eaf-9b18-4965-8da5-0f928389863f
+      id: 8ea51d16-1d64-40e6-8d3a-d5a459ffaf70
       version: -1
       name: QRadarFullSearch
```

Comment:
Also here , for some reason the subplaybook tasks returns to the name of the playbooks and not the name you gave to the task

Reply:
fixed